reactor-netty-http/src/main/java/reactor/netty/http/Http2SettingsSpec.java

```diff
@@ -241,5 +263,12 @@ public Builder pushEnabled(boolean pushEnabled) {
 			return this;
 		}
 		*/
+
+		@Override
+		public Builder minConnections(int minConnections) {
```

Comment:
I'm a big fan of where I put this, but just going fast in the config area.  I really wanted config to be able to test different scenarios.  Otherwise, I would have just hard-coded in `Http2Pool`.